src/frontend/src/components/ScrollToTop/ScrollToTop.jsx

```diff
@@ -58,7 +58,7 @@ export default function BackToTop(props) {
   const classes = useStyles();
   return (
     <>
-      <Toolbar id="back-to-top-anchor" className={classes.anchor} />
+      <Toolbar id="delete-in-the-future" className={classes.anchor} />
```

Comment:
If the toolbar does nothing and should be deleted, why not remove it now?

Reply:
There a lot of components with a high level of coupling, a mess on the banner, a lot of useEffects causing troubles. CSS that are causing child elements living outside and extremely far away from it's parents. So In the FUTURE I will check these components. I left some lines to help me comprehend the old ”logic”.

Reply:
I agree with @agarcia-caicedo, this is not a great solution.  What tends to happen with planning artifacts in source code is that they get lost.

A better solution here is to file an issue with all the details, then you can link it into a comment with `<!-- TODO: remove this as part of issue #1345 -->` or something.

@PedroFonsecaDEV could you file that issue now for me?

Reply:
Okay. Thanks